cloudbridge/cloud/providers/gce/services.py

```diff
@@ -774,10 +773,13 @@ def list(self, network=None, region=None, limit=None, marker=None):
                                      response.get('nextPageToken'),
                                      False, data=subnets)
 
-    def create(self, network, cidr_block, name=None, zone=None):
+    def create(self, network, cidr_block, name=None, zone=None, region=None):
         if not name:
             name = 'subnet-{0}'.format(uuid.uuid4())
-        region = self.provider.region_name
+        if region is not None and zone is not None:
```

Comment:
Does this require explicitly checking for "None" or can it be:

`if region and zone:
`

Reply:
N/A: this part is removed in the newer version #74.